letsencrypt/tests/cli_test.py

```diff
@@ -162,5 +167,73 @@ def test_no_accounts_email(self):
         self.assertEqual("other email", self.args.email)
 
 
+class DuplicativeCertsTest(unittest.TestCase):
+
+    def setUp(self):
+        # The stuff below is taken from renewer_test.py
+        self.tempdir = tempfile.mkdtemp()
+        self.cli_config = configuration.RenewerConfiguration(
+            namespace=mock.MagicMock(config_dir=self.tempdir))
+        os.makedirs(os.path.join(self.tempdir, "live", "example.org"))
+        os.makedirs(os.path.join(self.tempdir, "archive", "example.org"))
+        os.makedirs(os.path.join(self.tempdir, "configs"))
+        config = configobj.ConfigObj()
+        for kind in ALL_FOUR:
+            config[kind] = os.path.join(self.tempdir, "live", "example.org",
+                                        kind + ".pem")
+        config.filename = os.path.join(self.tempdir, "configs",
+                                       "example.org.conf")
+        config.write()
+        self.config = config
+        self.defaults = configobj.ConfigObj()
+        self.test_rc = storage.RenewableCert(
+            self.config, self.defaults, self.cli_config)
+        for kind in ALL_FOUR:
+            where = getattr(self.test_rc, kind)
+            os.symlink(os.path.join("..", "..", "archive", "example.org",
+                                    "{0}12.pem".format(kind)), where)
+            with open(where, "w") as f:
+                f.write(kind)
+            os.unlink(where)
+            os.symlink(os.path.join("..", "..", "archive", "example.org",
+                                    "{0}11.pem".format(kind)), where)
+            with open(where, "w") as f:
+                f.write(kind)
+
+        # Here we will use test_rc to create duplicative stuff
+
+    def tearDown(self):
+        shutil.rmtree(self.tempdir)
+
+    def test_find_duplicative_names(self):
+        from letsencrypt.cli import _find_duplicative_certs
```

Comment:
i'm surprised you don't have to `pylint: disable=protected-access` here; what's going on?